napari-workshops/installation.md

```diff
@@ -98,53 +98,43 @@ as well and you can skip to the next section.
 2. We use an environment to encapsulate the Python tools used for this workshop.
    This ensures that the requirements for this workshop do not interfere with
    your other Python projects. To create the environment (named
-   `napari-tutorial`) and install Python 3.9 in it, enter the following command:
+   `napari-workshop`) and install Python 3.11 and napari in it, enter the following command:
```

Comment:
This is to match the env.yml and using 3.11 is just better now with better error messages and bit faster.